plugins/org.jkiss.dbeaver.ui.app.standalone/src/org/jkiss/dbeaver/ui/app/standalone/DBeaverApplication.java

```diff
@@ -274,6 +278,12 @@ public Object start(IApplicationContext context) {
         }
         TimezoneRegistry.overrideTimezone();
 
+        if (RuntimeUtils.isWindows() &&
+            ModelPreferences.getPreferences().getBoolean(ModelPreferences.PROP_USE_WIN_TRUST_STORE_TYPE)
+        ) {
```

Comment:
It fits in a single line, let's not wrap it

```suggestion
        if (RuntimeUtils.isWindows() && ModelPreferences.getPreferences().getBoolean(ModelPreferences.PROP_USE_WIN_TRUST_STORE_TYPE)) {
```